packages/network-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [18.1.1]
+
+### Changed
+
+- Exported BuiltInNetworkClientId and CustomNetworkClientId strings ([#4247](https://github.com/MetaMask/core/pull/4247/files#diff-ea0aa37793484e278bd59970b7a6ce8380b75b723b34612081d402fb66261cfd))
```

Comment:
The change entry in the PR description seems more accurate here:

```suggestion
-  Export `BuiltInNetworkClientId` and `CustomNetworkClientId` ([#4247](https://github.com/MetaMask/core/pull/4247))
```

Calling these "strings" might be misleading, as these are actually types.

Also in this suggestion I've normalized the link to be just the PR, as is conventionally what we do for these entries

Reply:
Thank you. That makes a lot of sense.